frontend/wallet/src/render/components/SideBar.re

```diff
@@ -40,53 +40,56 @@ let make = () => {
     React.useContext(AddressBookProvider.context);
 
   <div className=Styles.sidebar>
-    <WalletQuery>
-      {response =>
-         switch (response.result) {
-         | Loading => React.string("...")
-         | Error(err) => React.string(err##message)
-         | Data(data) =>
-           <WalletList
-             wallets={Array.map(~f=Wallet.ofGraphqlExn, data##ownedWallets)}
-           />
-         }}
-    </WalletQuery>
-    <div className=Styles.footer>
-      <Link onClick={_ => setModalState(_ => Some("My Wallet"))}>
-        {React.string("+ Add wallet")}
-      </Link>
-    </div>
-    <AddWalletMutation>
-      {(mutation, _) =>
-         switch (modalState) {
-         | None => React.null
-         | Some(newWalletName) =>
-           <AddWalletModal
-             walletName=newWalletName
-             setModalState
-             onSubmit={name => {
-               let performMutation =
-                 Task.liftPromise(() =>
-                   mutation(~refetchQueries=[|"getWallets"|], ())
+    // TODO: Remove fetchPolicy="no-cache" after merge of
+    // https://github.com/apollographql/reason-apollo/pull/196
+
+      <WalletQuery fetchPolicy="no-cache" partialRefetch=true>
+        {response =>
+           switch (response.result) {
+           | Loading => React.string("...")
+           | Error(err) => React.string(err##message)
+           | Data(data) =>
+             <WalletList
+               wallets={Array.map(~f=Wallet.ofGraphqlExn, data##ownedWallets)}
+             />
+           }}
+      </WalletQuery>
+      <div className=Styles.footer>
+        <Link onClick={_ => setModalState(_ => Some("My Wallet"))}>
+          {React.string("+ Add wallet")}
```

Comment:
random idea - we might want to later use react-intl (or the equivalent reason library) and maybe have external contirbutors help us translate to different languages

Reply:
Eventually yeah that'd be dope, but I don't think that's a priority yet.

Reply:
We can search for React.string at that point and probably find most of these

Reply:
cool, def agree not a priority rn